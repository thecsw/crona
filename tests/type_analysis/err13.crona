foo : void() {}

main : void() {
    if (!"true") {}
    if ("true" || 7) {}
    if (99 && "false") {}
    if (foo || true) {}
}