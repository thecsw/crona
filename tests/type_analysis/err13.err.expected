FATAL [4,10]: Logical operator applied to non-bool operand
FATAL [5,9]: Logical operator applied to non-bool operand
FATAL [5,19]: Logical operator applied to non-bool operand
FATAL [6,9]: Logical operator applied to non-bool operand
FATAL [6,15]: Logical operator applied to non-bool operand
FATAL [7,9]: Logical operator applied to non-bool operand
Type Analysis Failed
